Modules/Core/Common/include/itkImageBase.h

```diff
@@ -509,6 +509,19 @@ class ITK_TEMPLATE_EXPORT ImageBase : public DataObject
     return index;
   }
 
+
+  /** \brief Returns the continuous index from a physical point
+   * \note This specific overload is easier to use, because it does not have template arguments. It just uses
+   * `itk::SpacePrecisionType`, both for the coordinates of the point and the index values.
+   *
+   * \sa Transform */
+  [[nodiscard]] ContinuousIndex<SpacePrecisionType, VImageDimension>
+  TransformPhysicalPointToContinuousIndex(const PointType & point) const
+  {
+    return TransformPhysicalPointToContinuousIndex<SpacePrecisionType>(point);
+  }
```

Comment:
Would it be useful to have two of such (simple, non-templated) overloads? One of `float` and one of `double`? Specifically:

    ContinuousIndex<float, VImageDimension>
    TransformPhysicalPointToContinuousIndex(const Point<float, VImageDimension> &) const;

    ContinuousIndex<double, VImageDimension>
    TransformPhysicalPointToContinuousIndex(const Point<double, VImageDimension> &) const;

So one from `Point<float, N>` to `ContinuousIndex<float, N>`, and one from `Point<double, N>` to `ContinuousIndex<double, N>`?  Overload resolution would then automatically choose the right one, no need for the user to specify template arguments. Right?

Reply:
The float one is rarely used, I believe. If someone wants an unusual thing, they can spell out the template parameters.

Reply:
I'm wondering, could all those `image->template TransformPhysicalPointToContinuousIndex<ContinuousValueType>(point)` calls inside ITK itself then replaced by simple non-templated `image->TransformPhysicalPointToContinuousIndex(point)` calls?